### PR TITLE
Use GPU before loading environment

### PR DESCRIPTION
**Pull request by emailweixu**
For some environment, it takes very long to load the environment. This PR change the code to use GPU before loading environments so `cluster_train` idle will not show those GPUs.

### CODE REVIEW

File: alf/environments/utils.py [hunk 1]
```diff
@@ -173,6 +174,10 @@ def create_environment(env_name='CartPole-v0',
 
     """
 
+    # Use GPU before loading environments so cluster_train idle will not show
+    # the GPU
+    tmp = torch.zeros((32, ))
```
Comment (hnyu) [inline alf/environments/utils.py:179]: I understand the motivation of this. But the comment seems unrelated to ALF?

Comment (emailweixu) [reply alf/environments/utils.py:179]: Updated the comment

**Review by hnyu — APPROVED** [review]
